src/Workspaces/Core/Portable/Options/GlobalOptionService.cs

```diff
@@ -178,6 +139,16 @@ public void SetOptions(OptionSet optionSet)
             RaiseEvents(changedOptions);
         }
 
+        public void RefreshOption(OptionKey optionKey, object newValue)
```

Comment:
is this a way to change one specific option rather than doing WithChangedOptions stuff?